docs/src/main/paradox/security/2018-09-05-denial-of-service-via-decodeRequest.md

```diff
@@ -0,0 +1,43 @@
+# CVE-2018-16131: Denial of Service via unlimited decoding with decodeRequest directive (“zip bomb”)
+
+## Date
+
+2018-09-05
+
+## Description
+
+Directives `decodeRequest` and `decodeRequestWith` which handle compressed request data did not limit the amount of uncompressed
+data flowing out of it. In combination with common request directives like `entity(as)`, `toStrict`, or `formField`, this can lead
+to excessive memory usage ultimately leading to an out of memory situation when highly compressed data is received
+(so-called "Zip Bomb").
+
+Any code that uses `decodeRequest` or `decodeRequestWith` is likely to be affected.
+
+## Severity
+
+The CVSS score of this vulnerability is 7.3 (High), based on vector
+[AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H/E:H/RL:W/RC:C](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H/E:H/RL:W/RC:C).
+
+Rationale for the score:
+
+ * A:H: Server runs into OOM, so availability is highly affected.
+ * E:H: It’s relatively simple to exploit.
+
+## Affected Versions
+
+All previosly released Akka HTTP versions are affected:
+
+ * 10.1.x <= 10.1.4
+ * 10.0.x <= 10.0.13
```

Comment:
this reads a bit weird to be honest... is this the established way to show version ranges we have adopted?
I think we did `- akka-http prior to `10.0.6` and  `2.4.11.2``